pom.xml

```diff
@@ -129,7 +129,7 @@
     <!-- Version used for internal directory structure -->
     <hive.version.short>1.2.1</hive.version.short>
     <derby.version>10.12.1.1</derby.version>
-    <parquet.version>1.8.2</parquet.version>
+    <parquet.version>1.10.0</parquet.version>
```

Comment:
@rdblue . To see the Jenkins result, could you resolve the dependency check failure with the following?
```
./dev/test-dependencies.sh --replace-manifest
```

Reply:
Unlike [last time](https://github.com/apache/spark/commit/26a4cba3ffaadf382ca14980378965704ccef9ab), it seems that this PR touches `commons-pool` dependency together? Can we avoid this?
```
-commons-pool-1.5.4.jar
+commons-pool-1.6.jar
```

Reply:
I excluded the commons-pool dependency from parquet-hadoop to avoid this. I also tested the latest Parquet release with commons-pool 1.5.4 and everything passes. I don't think it actually requires 1.6.